crates/extension_host/src/headless_host.rs

```diff
@@ -186,6 +187,17 @@ impl HeadlessExtensionStore {
                     );
                 })?;
             }
+            for (debug_adapter, meta) in &manifest.debug_adapters {
+                let schema_path = extension::build_debug_adapter_schema_path(debug_adapter, meta);
+
+                this.update(cx, |this, _cx| {
+                    this.proxy.register_debug_adapter(
+                        wasm_extension.clone(),
+                        debug_adapter.clone(),
+                        &extension_dir.join(schema_path),
+                    );
+                })?;
+            }
```

Comment:
We should also register debug locators: https://github.com/zed-industries/zed/blob/7a2593e5206fd8fe15be34cc8c195d3aa8ba97d1/crates/extension_host/src/extension_host.rs#L1379-L1382

Reply:
Done!